back-end/src/Plant/plant.controller.js

```diff
@@ -0,0 +1,17 @@
+const {Plant} = require('./plant.model')
+
+exports.getPlant = async (req,res) => {
+    try {
+        res.status(200).send(req.plant)
+    } catch (error) {
+        res.status(500).send({message:"Plant not found"})
+    }
+}
+
+exports.createPlant = async (res,req) => {
+    try {
+        const plant = new Plant(req.body)
+    } catch (error) {
+        
```

Comment:
You have no error handling here, if something goes wrong you'd have no way to know